pkg/server/server.go

```diff
@@ -90,6 +101,9 @@ func (server JSONRPCServer) ServeStream(_ context.Context, conn jsonrpc2.Conn) e
 	}
 	conn.Go(server.ctx, server.commandHandler)
 	<-conn.Done()
+
+	rollbar.Close()
```

Comment:
Would it be better to `defer` this at the start of the function? So that whatever happens we still `Close` rollbar.